src/containers/profile/ProfileSections/ManageUsers/ManageUsersSections/ProfileBox.jsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { Link, Redirect } from "react-router-dom";
+import { connect } from "react-redux";
+import { firestoreConnect } from "react-redux-firebase";
+import { compose } from "redux";
+import { updateProfile } from "../../../../../actions/authActions";
+import { Dropdown, Button, Row, Col, ListGroup, Image } from "react-bootstrap";
+import "./ProfileBox.css";
+import anonymousImage from "../../../../../images/anonymousImage.png";
+
+const ProfileBox = (props) => {
+  const { profile, auth } = props;
+  const photoSRC = profile.imageURL ? profile.imageURL : anonymousImage;
+
+  if (!auth.uid) {
+    return <Redirect to="/log-in" />;
+  }
+
+  return (
+    <div className="profileBox">
+      <Row>
+        <Col xs={8}>
+          <Link to={"/profiles/" + profile.id} key={profile.id}>
+            <ListGroup defaultActiveKey="none">
+              <ListGroup.Item action className="d-flex">
+                <Image
+                  src={photoSRC}
+                  alt="Profile"
+                  roundedCircle
+                  width="100"
+                  height="100"
+                  className="mr-5"
+                />
+                <div class="profileBoxInfo">
+                  <h4>{profile.fullName}</h4>
+                  <h6>
+                    {profile.jobTitle
+                      ? profile.jobTitle
+                      : "No job title available"}
+                  </h6>
+                  <h5>{profile.userType}</h5>
+                </div>
+              </ListGroup.Item>
+            </ListGroup>
+          </Link>
+        </Col>
+        <Col
+          xs={4}
+          className="d-flex flex-column pt-2 pb-2 justify-content-around"
+        >
+          {!profile.isBlocked ? (
```

Comment:
too many conditions here, hard to read. move this whole conditional logic out into a variable above the `return` and use a normal if condition.